folio-service-tools-dev/pom.xml

```diff
@@ -28,7 +28,6 @@
     <dependency>
       <groupId>org.folio.okapi</groupId>
       <artifactId>okapi-common</artifactId>
-      <version>2.26.0</version>
```

Comment:
why was this version removed? If we do not specify a version, it gets the latest version?

Reply:
I moved it to parent pom.xml, so now folio-service-tools-dev and folio-service-tools-test need to take version from the parent pom

Reply:
👍